api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -236,21 +236,36 @@ message PlanState {
   // The status of the plan.
   enum Status {
     UNKNOWN = 0;
+
+    // Plan execution has started. This will be sent a short time before AOS when the ground
+    // station has started preparing for the upcoming plan.
     STARTED = 1;
+
+    // Plan execution is in progress and has not yet reached the COMPLETE or FAILED state. This
+    // status may be sent multiple times.
```

Comment:
can consider adding more detail with something like `multiple times and acts as a heartbeat for the ground station.`